### ISSUE TITLE
[Feature Request] Make the Talk-to-ChatGPT Plugin UI More Compact

### ISSUE THREAD

**Issue by miztizm** [C-Nedelcu/talk-to-chatgpt#186]
Hello,

I'm an avid user of the Talk-to-ChatGPT Chrome plugin and appreciate the work that has gone into creating this useful extension. However, I've noticed that the current UI size is a bit intrusive and takes up a lot of screen space, making it hard to multitask. I would like to request an enhancement to make the plugin more compact.

**Current Behavior**

- When activated, the plugin window takes up a significant portion of the screen.
- Limits visibility of other active windows.
- Makes multitasking difficult.

**Desired Behavior**

- A more compact UI that maintains functionality while taking up less screen space.
- The ability to minimize or scale the window.

**Suggested Solution**

- Consider providing a settings option to resize the plugin window, or introduce a minimized version that can be expanded when needed.

Best regards

**Comment by C-Nedelcu** [comment]
hello,
I am inclined to agree with you. I have found this to be a problem in recent weeks.
I am going to be working on this and this will be published under v2.8.0
Thanks!

**Comment by C-Nedelcu** [comment]
So the first step towards this (probably the easy way) is I'm going to make 2.8.0 a lot more compact.
I've done that and will publish 2.8.0 on the stores shortly.

Taking it one step further would be to have a toggle button which basically would make it pretty much disappear (leaving just the toggle button visible) I might work on this later, or maybe @PxPerfectMike if you wanna give it a shot?

**Comment by PxPerfectMike** [comment]
Yeah that sounds great! I would love to do it. I think I can probably have it done by tomorrow evening.

**Comment by C-Nedelcu** [comment]
> Yeah that sounds great! I would love to do it. I think I can probably have it done by tomorrow evening.

awesome! here's an extra idea, maybe you could make it so when the widget is in minimized/reduced mode, the red/green/grey color still appears so that we have a visual of whats going on?  don't know if this makes sense

**Comment by PxPerfectMike** [comment]
Yeah that makes sense. I was thinking about adding a small minimize button to the main widget and then when it's minimized it will be a circle with the "chat state" colors in a ring around the edge of the circle. Clicking on the circle would return the widget to full size. Does that sound good?

**Comment by C-Nedelcu** [comment]
Yes - sounds like a good idea!
________________________________

**Comment by PxPerfectMike** [comment]
I just wanted to drop in here to let you know I still intend on finishing this feature, life has caught up with me and I got really busy. I'll be pushing it ASAP!

**Comment by miztizm** [comment]
Certainly, no worries at all! It's understandable that life can get busy sometimes. Thank you for letting us know about the status of the feature

**Comment by PxPerfectMike** [comment]
Hello, I submitted a PR for this feature request 3-weeks ago. Just wondering if there are any problems with the feature implementation. Thank you. Here is a link to the PR https://github.com/C-Nedelcu/talk-to-chatgpt/pull/228
